omero_figure/settings.py

```diff
@@ -5,4 +5,4 @@
 # https://www.openmicroscopy.org/
 #   site/support/omero5.2/developers/Web/CreateApp.html
 
-OMERO_FIGURE_VERSION = "2.0.0"
+OMERO_FIGURE_VERSION = "2.0.1"
```

Comment:
it is enough to provide version number using custom context processor, see webclient